setup.cfg

```diff
@@ -0,0 +1,89 @@
+[metadata]
+author = Christian Theune
+author_email = ct@flyingcircus.io
+classifiers =
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+description = Mirroring tool that implements the client (mirror) side of PEP 381
+long_description = file:README.md
```

Comment:
How do we get CHANGES.md now? It's nice to have the change log on https://pypi.org/p/bandersnatch
- It's not critical - Wonder if there is a way

Reply:
I added it under project_urls in the setup.cfg, that will add the link to the project_links section of the pypi page at https://pypi.org/project/bandersnatch/